parameters/navigation/global_costmap.yaml

```diff
@@ -30,7 +30,7 @@ recent_obstacles:
     occupied_to_default_time: 2.0
     observation_sources: base_laser rgbd_sensor
     base_laser:  {data_type: LaserScan, sensor_frame: /base_range_sensor_link, topic: /hero/base_laser/scan, observation_persistence: 0.0, inf_is_valid: true, obstacle_range: 2.5, marking: true, clearing: false, ray_trace_range: 3.0}
-    rgbd_sensor: {data_type: PointCloud2, sensor_frame: /head_rgbd_sensor_link, topic: /hero/ed/navigation/cloud, observation_persistence: 0.0, expected_update_rate: 1.0, min_obstacle_height: 0.0, max_obstacle_height: 0.5, obstacle_range: 10}
+    rgbd_sensor: {data_type: PointCloud2, sensor_frame: /head_rgbd_sensor_link, topic: /hero/ed/navigation/cloud, observation_persistence: 0.0, expected_update_rate: 0.0, min_obstacle_height: 0.0, max_obstacle_height: 0.5, obstacle_range: 10}
```

Comment:
It may be convenient to set the expected update rate to 0.0 so it doesn't cause warnings in simulation, however there should be a warning on the real robot.

Reply:
@LoyVanBeek @PetervDooren I think this is a difficult problem because you do not want to have multiple param files..... but you also do not want have all these warnings in simulation.

Reply:
You could specify the plugins: 

``` yaml
plugins:
    # Obstacles
    - {name: ed_occupancy_grid,                 type: "costmap_2d::StaticLayer"}
    - {name: recent_obstacles,                  type: "costmap_2d::ObstacleLayer"}
    - {name: robot_footprint,                   type: "costmap_2d::FootprintLayer"}
    - {name: configuration_space,               type: "costmap_2d::InflationLayer"}
```

in the launch file with a conditional based on whether you are in simulation or not

Reply:
Rather than the plugins I specified the observation sources  to be just the base laser if we are working in simulation. These changes were made in the robot launch files repository. See PR https://github.com/tue-robotics/robot_launch_files/pull/51